autopts/wid/gap.py

```diff
@@ -1707,7 +1707,9 @@ def hdl_wid_301(params: WIDParams):
     stack = get_stack()
 
     # GAP/PADV/PASE/BV-02-C starts PA Sync in WID 302 which comes first
-    if params.test_case_name not in ['GAP/PADV/PASE/BV-02-C']:
+    if params.test_case_name not in ['GAP/PADV/PASE/BV-02-C',
+                                     'GAP/PADV/PASE/BV-04-C',
+                                     'GAP/PADV/PASE/BV-06-C']:
```

Comment:
Just adding the same comment here as I did for the PR adding this list in the first place:
IMO it makes more sense to keep track of the PA sync state in the `stack` and then check if we have a PA sync at this point or not, and if not, then we establish the sync. 

I assume that the newly added tests don't have a WID call earlier to establish the PA sync?

Reply:
I'd extend settings with PA_SYNCING and eg CONN_INITIATING etc and start using those,  however this requires for UT to properly update settings (yet, we could enforce those when issuing command that should result in settings change)

this way state doesn't have to be in stack (which is IUT specific) but in generic code
this is a slightly bigger refactor though